src/index.ts

```diff
@@ -50,17 +50,17 @@ import { InitAuth0, SignInWithAuth0 } from './instance';
 import version from './version';
 import { getConfig, getLoginUrl, ConfigParameters } from './config';
 
-let instance: SignInWithAuth0;
+let instance: SignInWithAuth0 & { sessionCache: SessionCache };
```

Comment:
Can we define a type alias for this?

Reply:
It's only used internally in this file, creating an alias seems overkill